source/onboard/mattermost-limits.rst

```diff
@@ -20,17 +20,19 @@ Mattermost Free limits
 **Platform limits**
 
 - Maximum 1 team. Any additional teams created during a trial or on a paid plan will be archived.
-- 10 GB file storage across the platform, with 100 MB upload limit. The oldest files that exceed this limit will be archived.
-- Unlimited plugins can be installed but only 5 can be enabled at one time. Core plugins such as Boards, Calls, and Playbooks do not count toward this limit.
+- 10 GB file storage across the platform, with 100 MB upload limit. The oldest files that exceed this limit will be archived. Card attachments are counted toward this limit.
 
 **Channels limits**
 
 - 10,000 message history limit. The oldest messages that exceed this limit will be archived.
 
+**Calls limits**
+
+- Up to 8 participants per group call.
```

Comment:
@tao - is this per individual call? Can there be multiple calls of up to 8 people at any given time?